src/gui/src/components/explorer-grid/overview-sidebar/suggested-queries.tsx

```diff
@@ -0,0 +1,130 @@
+import { useEffect, useState } from 'react'
+import { ChevronRight, Search, CornerDownRight } from 'lucide-react'
+import { Button } from '@/components/ui/button.tsx'
+import { Query } from '@vltpkg/query'
+import { QueryToken } from '@/components/query-bar/query-token.tsx'
+import { useGraphStore } from '@/state/index.ts'
+import { AnimatePresence, motion } from 'framer-motion'
+
+interface SuggestedQuery {
+  name: string
+  description: string
+  query: string
+}
+
+export const queries: SuggestedQuery[] = [
+  {
+    name: 'Outdated',
+    description: 'Find packages that are outdated',
+    query: ':outdated(major)',
+  },
+  {
+    name: 'Unmaintained',
+    query: ':unmaintained',
+    description: 'Find packages that are unmaintained',
+  },
+  {
+    name: 'System access',
+    query: ':has(:fs) :has(:env)',
+    description:
+      'Find packages with access to filesystem and environment variables',
```

Comment:
I think the correct description for this query would be:

```suggestion
      'Find packages that contains dependencies with access to filesystem or environment variables',
```

if you want a query that actually behaves like the description then we'd need to change `:has()` by `:is()`, e.g:

```
:is(:fs):is(:env)
```